python/lsst/afw/table/multiMatch.py

```diff
@@ -259,7 +259,7 @@ def __len__(self):
         return len(self.ids)
 
     def __iter__(self):
-        return self.ids
+        yield from self.ids
```

Comment:
I would have expected `return iter(self.ids)`.

Reply:
Aren't they essentially equivalent?  I'm happy to do either.

Reply:
I've implemented your recommendation.

Reply:
They may well be equivalent, but given that the method is called `__iter__`, I'm wired to expect to see `return iter(...)`.